manage_projects.rb

```diff
@@ -31,7 +31,7 @@
 def add_or_update_project(action=nil)
   @factory = ProjectFactory.new
   if action == nil
-    print "List, add or update project(s) (list/add/update/validate)? "
+    print "List, add, or update project(s) (list/add/update/validate)? "
```

Comment:
Don't we want specific actions for enabling/ disabling?

Reply:
Are there any major benefits to this over having it be lumped in with the rest of the attributes?

Reply:
Maybe I'm missing something, but I thought that was the main purpose of issue #45 (and mentioned in #38) ? Otherwise we've effectively already met the requirement by letting the user manage if a project is active by setting the end date and start date.